src/pages/LandingPage.tsx

```diff
@@ -55,7 +55,6 @@ const LandingPage = () => {
           MISSION
         </span>
       </h1>
-
```

Comment:
This may have been an accident, but please undo this change (i.e. re-add this blank line). This will clean up the diff and show only 2 changed files!